File: src/main/java/com/lmptech/intune/data/repositories/ChatRepository.java
package com.lmptech.intune.data.repositories;

import com.lmptech.intune.data.models.ChatModel;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChatRepository extends MongoRepository<ChatModel, String> {
}

File: src/main/kotlin/com/x64tech/intune/respositories/ChatRepository.kt
package com.x64tech.intune.respositories

import com.x64tech.intune.entites.ChatEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface ChatRepository: JpaRepository<ChatEntity, UUID> {
}